jetson/power_logging/pytorch/layer_execution_time.py

```diff
@@ -115,5 +115,6 @@ def get_layer_execution_time(model_name, input_shape, num_inference_cycles) -> N
         model(x)
         all_cycle_measurements[f"cycle_{i}"] = layer_time_dict
 
-    with open(f"{datetime.datetime.now().strftime('%Y_%m_%d_%H:%M:%S')}_{model_name}_inference_trace_{num_inference_cycles}_cycles.json", "w") as f:
+    with open(f"{datetime.now().strftime('%Y_%m_%d_%H:%M:%S')}_{model_name}_inference_trace_{num_inference_cycles}_cycles.json", "w") as f:
```

Comment:
I would recommended avoiding adding timestamps to save the name of trace file. It will create a conflict when running experiments multiple times for same network and using DagsHub for data versioning.

`{model_name}_layerwise_latency.json` should suffice.

Reply:
For reference, take  a look at how the raw data is saved for TensorRT profiling `trt_profiling` folder: https://dagshub.com/fuzzylabs/edge-vision-power-estimation/src/develop/jetson/power_logging/raw_data/prebuilt_models/alexnet